Dockerfile

```diff
@@ -1,8 +1,15 @@
+FROM       golang:alpine as builder
+
+RUN apk --no-cache add curl git make perl
+RUN curl -s https://glide.sh/get | sh
+RUN git clone https://github.com/dcu/mongodb_exporter.git /go/src/github.com/dcu/mongodb_exporter
```

Comment:
does this need to be an absolute path?

Reply:
I have to confess that I wasn't sure so I err'd on the side of caution / went with what worked first.

Reply:
Thanks for reviewing this so quickly!